pkgs/development/python-modules/pdfrw2/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "pdfrw2";
+  version = "0.5.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-5qnMq4Pnaaeov+Lb3fD0ndfr5SAy6SlXTwG7v6IZce0=";
+  };
+
```

I think I spoke a bit soon. Doesn't this package rely on some dependencies?

From these files, it seems to need at least these deps:

- https://github.com/sarnold/pdfrw/blob/master/requirements.txt
- https://github.com/sarnold/pdfrw/blob/master/pyproject.toml

```suggestion

propagatedBuildInputs = [
  pillow
  reportlab
  pycryptodomex
];

nativeBuildInputs = [
  setuptools
];

```

---

Oh sorry you're right, it should be fixed now.